App/Redux/TextileEventsRedux.ts

```diff
@@ -56,30 +48,35 @@ interface TextileEventsState {
     readonly state: NodeState
     readonly error?: string
   }
-  readonly online: boolean
 }
 
 export const initialState: TextileEventsState = {
   nodeState: {
-    state: NodeState.nonexistent
-  },
-  online: false
+    state: NodeState.stopped
+  }
 }
 
 export function reducer(state: TextileEventsState = initialState, action: TextileEventsActions): TextileEventsState {
   switch (action.type) {
-    case getType(actions.newNodeState):
-      return { ...state, nodeState: {state: action.payload.state }}
+    case getType(actions.nodeStarted):
+      return { ...state, nodeState: { state: NodeState.started } }
+    case getType(actions.nodeStopped):
+      return { ...state, nodeState: { state: NodeState.stopped } }
     case getType(actions.nodeOnline):
-      return { ...state, online: true }
+      return { ...state, nodeState: { state: NodeState.online } }
+    case getType(actions.nodeFailedToStart):
+    case getType(actions.nodeFailedToStop):
+      const { error } = action.payload
+      return { ...state, nodeState: { ...state.nodeState, error } }
     default:
       return state
   }
 }
 
 export const TextileEventsSelectors = {
   nodeState: (state: RootState) => state.textile.nodeState,
-  online: (state: RootState) => state.textile.online
+  started: (state: RootState) => state.textile.nodeState.state === NodeState.started || state.textile.nodeState.state === NodeState.online,
```

Comment:
I'm modeling `online` as another `NodeState`, so when the node is "started" it can be in either a `started` or `online` state... This selector makes it easy to ask if the node is "started".